CRM/Upgrade/Snapshot.php

```diff
@@ -44,8 +44,20 @@ class CRM_Upgrade_Snapshot {
   public static function getActivationIssues(): array {
     if (static::$activationIssues === NULL) {
       $policy = CRM_Utils_Constant::value('CIVICRM_UPGRADE_SNAPSHOT', 'auto');
+      static::$activationIssues = [];
+
+      $version = CRM_Utils_SQL::getDatabaseVersion();
+      if (stripos($version, 'mariadb') !== FALSE) {
+        // MariaDB briefly (10.6.0-10.6.5) flirted with the idea of phasing-out `COMPRESSED`. By default, snapshots won't work on those versions.
+        // https://mariadb.com/kb/en/innodb-compressed-row-format/#read-only
+        $roCompressed = CRM_Core_DAO::singleValueQuery('SELECT @@innodb_read_only_compressed');
```

Comment:
Hmm this variable doesn't exist on mariadb 10.3 so it's a hard crash. From the docs it looks like it doesn't exist until 10.6?

Reply:
PR: https://github.com/civicrm/civicrm-core/pull/27464